src/Sql/SqlPgsql.php

```diff
@@ -118,6 +118,11 @@ public function listTables()
         return array_filter($tables);
     }
 
+    public function listTablesQuoted()
```

Comment:
You can remove this + the same method in Sqlite now that this is in the base class.